.gitignore

```diff
@@ -8,3 +8,4 @@ releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
 .vagrant/
+.vscode/
```

Comment:
Would you mind reverting this?  We don't include editor- or IDE-specific ignores in our project `.gitignore`.  If you're seeing editor files in your repository, you can configure a [global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for yourself.

Reply:
Ok, `.gitignore` is reverted. Not sure my current `Signed-off-by` is correct or not after this change.

Reply:
It looks like you updated the PR description but not the commit message.  Can you update the commit message? `git commit --amend` should let you edit it.

Reply:
hmmm, I typed `git commit -s --amend --no-edit` before `git push -f` and edited the PR desc minutes ago.

Reply:
`--no-edit` means git won't prompt you to update the commit message.  Can you update the commit message?

Reply:
wow, great thanks, learnt new knowledge. It looks compliant now.